test/other/codesize/test_codesize_hello_dylink.size

```diff
@@ -1 +1 @@
-9347
+9760
```

Comment:
Is calloc really 413 bytes larger than malloc? That seems surprising.

Reply:
I don't think its a huge deal the JS size went down a few bytes the native size went up a few bytes.

Also this is the only code size test in the test suite that was effected, so it only appear to effect the dyanmic linking using case.. at least that is the only test where any change occured.

Reply:
Hmm, but it is more than a few bytes: 413 bytes seems quite a lot for calloc to add over malloc. It should just have a loop that writes zeroes? I'm worries something else is going on there.

(The JS change in the other direction is only 9 bytes, which does make sense.)